__tests__/components/common/Button.test.tsx

```diff
@@ -9,4 +9,52 @@ describe('Button.tsx', () => {
 
     expect(screen.getByText(expectedText)).toBeInTheDocument();
   });
+
+  describe('when "aurora" theme is selected', () => {
+    it('should have correct classNames', () => {
+      render(
+        <Button onClick={jest.fn()} theme='aurora'>
+          hello button
+        </Button>
+      );
+
+      expect(screen.getByRole('button')).toHaveClass(
+        'border-ligh-gray text-white bg-gradient-to-r from-sky-500 to-fuchsia-500'
+      );
+    });
+  });
+
+  describe('when no theme is selected', () => {
+    it('should have default classNames', () => {
+      render(<Button onClick={jest.fn()}>hello button</Button>);
+
+      expect(screen.getByRole('button')).toHaveClass(
+        'bg-carbon-gray border-ligh-gray text-white'
```

Comment:
It should be `border-light-gray`

It originates from your tailwind.config.js

Reply:
ohhh got it thanks, i did not noticed!